item.rb

```diff
@@ -7,14 +7,14 @@ def initialize(publish_date, id = nil, archived: false)
     @publish_date = publish_date
```

Comment:
published date should be an object
`@publish_date = DateHandler.from_string(publish_date)`

Reply:
Hi, Thanks for the review. This is being handled while creating a new Object in the Music Album Manager, there we can check if the Date format is correct and if it is correct we convert it into a date object.

![image](https://user-images.githubusercontent.com/73714615/170291514-a36874e2-3add-4660-a600-df1be187a003.png)